PICMI_Python/fields.py

```diff
@@ -1124,3 +1124,29 @@ def add_refined_region(self, level, lo, hi, refinement_factor=[2,2,2]):
             Defaulting to [2,2,2] (relative to next lower level)
         """
         self.refined_regions.append([level, lo, hi, refinement_factor])
+        
+class PICMI_LoadInitialField(_ClassWithInit):
+    '''Class to read in external field from file
+
+    Parameters
+    ----------
+    read_fields_from_path: string
+        Path to file with field data
+
+    load_B: bool, default=True
+        If False, do not load magnetic field
+
+    load_E: bool, default=True
+        If False, do not load electric field
+    '''
+    def __init__(self, read_fields_from_path,load_B=True, load_E=True, **kw) :
+        self.load_B = load_B
+        self.load_E = load_E
+        self.read_fields_from_path = read_fields_from_path
```

Comment:
This routine should call "self.handle_init(kw)". See the other classes defined here as examples.